resources/webroot/index.js.mustache

```diff
@@ -2554,12 +2566,23 @@ function getPokestopPopupContent (pokestop) {
   const isActiveLure = lureExpireDate >= now
   const isPoweredUp = powerUpEndDate >= now && parseInt(pokestop.power_up_level) > 0
 
+  let regex = /\S[\s\S]{0,23}\S(?=\s|$)/g
   let content = '<div class="text-center">'
-  let pokestopName
+  let pokestopName = ''
   if (pokestop.name === null || pokestop.name === '') {
-    pokestopName = 'Unknown Pokestop Name'
+    pokestopName = i18n('map_unknown_pokestop')
   } else {
-    pokestopName = pokestop.name
+    if (pokestop.name.length <= 26) {
+      pokestopName = pokestop.name
+    } else {
+      let m
+      let choppedName = new Array();
+
+      while ((m = regex.exec(pokestop.name)) !== null) {
+        choppedName.push(m[0]);
+      }
+      pokestopName = choppedName.join('<br>')
+    }
   }
 
   content += '<span class="text-nowrap"><h6><b>{{#perm_admin}}<a href="#" onclick="editPokestopName(\''+pokestop.id+'\',\''+pokestop.name+'\')">{{/perm_admin}}' +
```

Comment:
I think we can keep the old way for pokestop/gym name and remove `text-nowrap` for name. maybe use 2 cols for team icon and name :)